BitwardenShared/Core/Vault/Services/ExportVaultServiceTests.swift

```diff
@@ -274,6 +274,85 @@ final class ExportVaultServiceTests: BitwardenTestCase {
         )
     }
 
+    /// Test that no ciphers are filtered when restrictedTypes is empty
+    ///
+    func test_fileContent_restrictedTypes_empty() async throws {
+        let fileType = ExportFileType.json
```

Comment:
⛏️ Is this variable necessary? Couldn't we just pass `ExportFileType.json` to the `format:` parameter? Same applies to other tests.